pkg/arvo/sur/chat-view-imports.hoon

```diff
@@ -0,0 +1,4 @@
+/-  *permission-store, *permission-hook, *group-store, *permission-group-hook
+|%
+--
```

Comment:
Ha, never seen that pattern before.  Is this only used in one place?  If so, is it really a necessary bit of indirection?  Makes it a bit harder to see where stuff is coming from.  If you're worried about 80 cols, I think you can split `/-` across multiple lines.

Reply:
Pfft, `|% --` definitely shouldn't be valid hoon - if just the `/-` won't work, maybe try `.`?

Reply:
If you look at Ford's parser for /- runes, it does not support multiple lines. It also does not support multiple /- statements per file. In the interest of being below 80 characters per line, I'm going to keep this the same for now. Eventually, it would be nice for /- to support gaps with an end to the list as ==.

https://github.com/urbit/urbit/blob/2bb3c5a551d473fa94f555c1206f3fac88542ae9/pkg/arvo/sys/vane/ford.hoon#L893

Reply:
I'm pretty sure `gaw` includes a newline.  This seemed to work in a generator:

```
/-  a,
    b
```

Reply:
ah, great. trying that now.

Reply:
that is VERY irregular syntax for hoon :(

Reply:
okay, that does indeed work. removed the sur/chat-view-imports file and added the syntax you recommended.